.changeset/forty-pens-obey.md

```diff
@@ -0,0 +1,5 @@
+---
+"@graphiql/plugin-code-exporter": patch
+---
+
+chore(@graphiql/plugin-code-exporter): Fix Typo
```

Comment:
As it's just the README, we don't need a changeset for that (we don't need to publish a new version just because of a typo 😅 )

Reply:
But otherwise people reading the README on npm won’t get the update

Reply:
good point @LekoArts - especially because it's currently a standalone workspace, it won't get another release until there's another changeset to just this plugin